libsnark/gadgetlib1/gadgets/hashes/anemoi/anemoi_components.tcc

```diff
@@ -34,28 +34,23 @@ namespace libsnark
 //
 // where A =(0, const_a, 0), B=(0, 1, 0) and C =(-const_b, 0, 1)
 
-template<typename FieldT, size_t generator>
-flystel_Q_gamma_prime_field_gadget<FieldT, generator>::
+template<typename ppT, class parameters>
+flystel_Q_gamma_prime_field_gadget<ppT, parameters>::
     flystel_Q_gamma_prime_field_gadget(
-        protoboard<FieldT> &pb,
-        const linear_combination<FieldT> &input,
-        const pb_variable<FieldT> &output,
+        protoboard<libff::Fr<ppT>> &pb,
```

Comment:
I opted to use `ppT` as a template parameter everywhere instead of `FieldT`, as it is more general (from `ppT` you can get `FieldT`, but not the other way around). Would you suggest to still use `FieldT` as a template parameter whenever possible?

Reply:
No, fine to use ppT everywhere.  Just wanted to make sure of the reasoning.